apps/modernization-ui/src/apps/search/event/components/InvestigationSearch/InvestigationGeneralFields.tsx

```diff
@@ -233,15 +233,27 @@ export const InvestigationGeneralFields = ({ form }: InvestigationGeneralAccordi
                     control={form.control}
                     name="createdBy"
                     render={({ field: { onChange } }) => (
-                        <UserAutocomplete id="createdBy" label="Event created by user" onChange={onChange} />
+                        <UserAutocomplete
+                            asValue={(value) => value}
+                            asText={(value) => value?.name}
```

Comment:
suggestion: If you find yourself going to every place a component is used to make a change, try to make that change in the component itself.  Also, is this even needed?  I thought we added that `Optional` type to be able to remove this requirement.

Reply:
I just added the optional type back in so all good now